tests/support/helpers.py

```diff
@@ -1153,7 +1153,7 @@ def skip_if_not_root(func):
     if RUNTIME_VARS.PYTEST_SESSION:
         setattr(func, '__skip_if_not_root__', True)
 
-    if not sys.platform.startswith('win'):
+    if not salt.utils.platform.is_windows() and hasattr(os, 'getuid'):
```

Comment:
This seems weird. Rather than working around the os module not having `getuid` we should get to the root of what's causing that. Maybe sync with @s0undt3ch on that.

Reply:
Yeah, let's not work around that.